docs/source/getting_started/index.rst

```diff
@@ -1,24 +1,113 @@
-Getting started
+Getting Started
 ===============
 
-.. include:: install.rst
+Installation
+^^^^^^^^^^^^
 
-Next steps
-^^^^^^^^^^
+The simplest installation is through `pip <https://pypi.org/project/dowhy/>`__ or conda:
 
-DoWhy is based on a simple unifying language for causal inference, unifying two powerful frameworks,
-graphical causal models and potential outcomes. It uses graph-based criteria and do-calculus for modeling assumptions
-and identifying a non-parametric causal effect.
+.. tab-set-code::
 
-For estimation, it switches to methods based primarily on potential outcomes and the best place to learn more about
-this is the user guide's :doc:`../user_guide/effect_inference/index` chapter. Alternatively, learn about its basic
-features in :doc:`../example_notebooks/dowhy_simple_example`.
+    .. code-block:: pip
 
-For root cause analysis features and structural analysis it uses graphical causal models and we recommend starting
-with :doc:`../user_guide/gcm_based_inference/index` in the user guide or check out
-:doc:`../example_notebooks/gcm_basic_example`.
+        pip install dowhy
 
-Finally, there's further resources available:
+    .. code-block:: conda
+
+        conda install -c conda-forge dowhy
+
+Further installation scenarios and instructions can be found at :doc:`install`.
+
+"Hello causal inference world"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this section, we will show the "Hello world" version of DoWhy. DoWhy is based on a simple unifying language for
+causal inference, unifying two powerful frameworks, namely graphical causal models (GCM) and potential outcomes (PO).
+It uses graph-based criteria and do-calculus for modeling assumptions and identifying a non-parametric causal effect.
+
+To get you started, we introduce two features out of a large variety of features DoWhy offers.
+
+Effect inference
+----------------
+
+For effect estimation, DoWhy switches to methods based primarily on potential outcomes. To do it, DoWhy offers a
```

Comment:
we also use causal graphs for effect identification step. Since the two frameworks are mentioned above, I guess we can focus on an end-user's perspective here.

Maybe rephrase it to, "For effect inference, DoWhy offers a 4-step recipe consisting of modeling a causal graph, identification, estimation, and refutation. The four steps are designed to explicitly bring out the assumptions in a causal analysis and then validate them in the refutation step. DoWhy supports estimation of both average and conditional causal effects.

Reply:
This actually not a new formulation, but an existing one:
- https://www.pywhy.org/dowhy/v0.8/user_guide/causality_intro.html?highlight=estimation
- https://www.pywhy.org/dowhy/main/getting_started/index.html#next-steps

I do get your point though. Alternative suggestion: Kailash and I have already started working restructuring our User guide by shifting away the focus from the different APIs we currently have to specific tasks. This is not easy, but I think once we have that, we can also apply the same technique here in the Getting started section. Can we post-pone the improvement until then?

Reply:
Sounds good. We can postpone this.